eclair-core/src/main/scala/fr/acinq/eclair/channel/Channel.scala

```diff
@@ -234,6 +234,7 @@ class Channel(val nodeParams: NodeParams, val wallet: EclairWallet, remoteNodeId
               closing.localCommitPublished.foreach(doPublish)
               closing.remoteCommitPublished.foreach(doPublish)
               closing.nextRemoteCommitPublished.foreach(doPublish)
+              closing.customRemoteCommitPublished.values.foreach(doPublish)
```

Comment:
We should have this line of code for consistency, it will never be used, right (since we're not persisting the custom remote commits)?

Reply:
Yep!